docs/keys/at.md

```diff
@@ -4,20 +4,23 @@
 Return whatever value is at a key, regardless
 if it's a function or a compute.
 
+The default behavior in Stache lookups is to execute functions to get their return values when following a dotted path.  `@` before a token in a path `@A.B` prevents A from being called for its return value (so 'B' is evaluated from the properties of A itself). Likewise, `@` elsewhere in a path like `A@B` prevents B from being called for its return value and returns whatever B itself is on the return from A.
```

Comment:
I don't think the `when following a dotted path` clause is necessary in the first sentence. It seems to imply that `{{#if func}}...{{/if}}` won't cause `func` to be called, when it will.